src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -2428,124 +2429,137 @@ msgid "submission confirmation title"
 msgstr "Paginatitel inzendingsbevestiging"
 
 #: openforms/config/models/config.py:70
+#, fuzzy
+#| msgid ""
+#| "The content of the confirmation page title. You can (and should) use the "
+#| "'public_reference' variable so the users have a reference in case they "
+#| "need to contact the customer service."
```

Comment:
Fuzzy translations are not actually used/picked up. Did you use rosetta for the translations?